### PR TITLE
Add /void, /notes, /consolidate_invoices

### PR DESCRIPTION
**Pull request by adam-invoiced**
On this:
- unsure of order to place new things in Invoicing section or if that section is now large enough to be split
- unsure whether notes section code examples are as desired; I preserved some methods on invoice object

### CODE REVIEW

File: source/includes/_invoicing.md.erb [hunk 1]
```diff
@@ -680,6 +715,118 @@ This endpoint retrieves a list of files attached to a specific invoice.
...
+### HTTP Request
+
+`POST /customers/:id/consolidate_invoices`
+
+### Attributes
+
+Parameter | Type | Description
+--------- | ---- | -----------
+**cutoff_date** | *timestamp* | Optional cutoff date
+
```
Comment (jaredtking) [inline source/includes/_invoicing.md.erb]: Can you document that an error happens here when there are no invoices to consolidate, or the customer does not have consolidation enabled?

File: source/includes/_invoicing.md.erb [hunk 2]
```diff
@@ -787,3 +934,299 @@ Parameter | Description
 **start_date** *timestamp* | Restricts the results to invoices *on or after* the given timestamp
 **end_date** *timestamp* | Restricts the results to invoices *on or before* the given timestamp
 **updated_after** *timestamp* | Only gets records updated after the given timestamp
+
+## Create a note
```
Comment (jaredtking) [inline source/includes/_invoicing.md.erb]: Let's put notes in a separate section after files. Notes can be attached to customers, and not just invoices.

**Comment by jaredtking** [comment]
Can you rebase against master?

**Comment by adam-invoiced** [comment]
should be good

**Pull request merged**